src/core/evaluator.js

```diff
@@ -459,14 +459,14 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
         return;
       }
 
-      var useNativeImageDecoder = !this.options.disableNativeImageDecoder;
+      var useNativeImageDecoder = this.options.nativeImageDecoderSupport;
       // If there is no imageMask, create the PDFImage and a lot
       // of image processing can be done here.
       var objId = 'img_' + this.idFactory.createObjId();
       operatorList.addDependency(objId);
       args = [objId, w, h];
 
-      if (useNativeImageDecoder &&
+      if (useNativeImageDecoder !== 'none' &&
```

Comment:
With the comments above taken into account, this line should be:
`if (nativeImageDecoderSupport !== NativeImageDecoding.NONE &&`